app/utils/callback/lava.py

```diff
@@ -615,16 +677,25 @@ def add_tests(job_data, job_meta, lab_name, db_options,
         _store_lava_json(job_data, meta)
         groups = []
         cases = []
+        start_log_line = 0
+        log = yaml.load(job_data["log"], Loader=yaml.CLoader)
+        end_lines_map = {}
 
         for suite_name, results in job_data["results"].iteritems():
             if suite_name == "lava":
                 _add_login_case(meta, results, cases, 'auto-login-action')
+                start_log_line = _get_log_line_number(log,
+                                                      LOGIN_CASE_END_PATTERN
+                                                      ) or 0
             else:
                 suite_name = suite_name.partition("_")[2]
                 group = dict(meta)
                 group[models.NAME_KEY] = suite_name
-                _add_test_results(group, results)
+                end_lines_map[group[models.NAME_KEY]] = []
```

Comment:
Following my previous comment, that would mean leaving the call to `_add_test_results(group, results)` unchanged but add a call to a new function e.g. `_get_line_numbers(group, results)` and append that to `end_lines_map`.